static/js/popovers.js

```diff
@@ -547,6 +547,14 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
     });
 
+    $('body').on('click', '.mark_stream_as_read', function (e) {
+        var stream = $(e.currentTarget).parents('ul').attr('data-name');
+        //var topic = $(e.currentTarget).attr('data-topic-name');
```

Comment:
Trying queries such as `UserMessage.objects.filter(message__topic__iexact=topic)` is failing tests with the message "Relation fields do not support nested lookups" 

changing this to UserMessage.objecs.filter(message__subject__iexact=topic) seems to do the trick, if I'm not mistaken